test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -1456,7 +1453,7 @@ private int runAndGetExitStatus(ProcessBuilder pb) {
 			int p = 0;
 			for (String a: actualArguments) {
 				if (a.startsWith(m)) {
-					argPositions.put(m, p);
+					argPositions.put(m, new Integer(p));
```

Comment:
`Integer.valueOf(int)` is usually a better choice as it avoids allocation for ints in the cached range: https://docs.oracle.com/javase/9/docs/api/java/lang/Integer.html#valueOf-int-

This is fine here though.

Reply:
Cool.  Thanks.